File: scala-gatling/src/gatling/scala/org/openapitools/client/model/RequestTimeFilterPostcodes.scala

package org.openapitools.client.model


case class RequestTimeFilterPostcodes (
    _departureSearches: Option[List[RequestTimeFilterPostcodesDepartureSearch]],
    _arrivalSearches: Option[List[RequestTimeFilterPostcodesArrivalSearch]]
)
object RequestTimeFilterPostcodes {
    def toStringBody(var_departureSearches: Object, var_arrivalSearches: Object) =
        s"""
        | {
        | "departureSearches":$var_departureSearches,"arrivalSearches":$var_arrivalSearches
        | }
        """.stripMargin
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/RequestTimeFilterPostcodes.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.RequestTimeFilterPostcodesArrivalSearch
open traveltimeplatform.Model.RequestTimeFilterPostcodesDepartureSearch

module RequestTimeFilterPostcodes = 

  //#region RequestTimeFilterPostcodes


  type RequestTimeFilterPostcodes = {
    DepartureSearches : RequestTimeFilterPostcodesDepartureSearch[];
    ArrivalSearches : RequestTimeFilterPostcodesArrivalSearch[];
  }
  //#endregion
  